### PR TITLE
UCPKN-2600: Accessibility statement link in the footer.

### PR DESCRIPTION
**Pull request by peterserfozo**

### CODE REVIEW

**Review by msnassar — COMMENTED** [review]
It has been tested on UCPKN. Works as expected. However, it would be nice if the oe_corporate_blocks module do the preprocess for the accessibility link (the same way it does for the site_owner) `oe_corporate_blocks_preprocess_set_site_owner`. By doing this, we won't need to copy  `_oe_whitelabel_preprocess_accessibility_link` from the oe_theme and both oe_theme and oe_whitelabel will have same preprocess. I would like you know your thoughts?

**Review by dxvargas — DISMISSED** [review]

**Comment by brummbar** [comment]
@msnassar I like that, let's do it there. I'll check with core team.

**Comment by brummbar** [comment]
@msnassar @peterserfozo confirmed that we can move this to corporate_blocks module as it's already depending on site_info.

**Review by drishu — DISMISSED** [review]

**Review by drishu — DISMISSED** [review]

File: tests/src/Kernel/FooterBlockTest.php [hunk 1]
```diff
@@ -23,6 +23,8 @@ class FooterBlockTest extends SparqlKernelTestBase {
     'oe_whitelabel_helper',
     'rdf_skos',
     'system',
+    'ui_patterns',
+    'ui_patterns_library',
```
Comment (brummbar) [inline tests/src/Kernel/FooterBlockTest.php:27]: Why this?

File: templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig [hunk 2]
```diff
@@ -1,4 +1,14 @@
 {% extends "@oe_whitelabel/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig" %}
+
+{% block accessibility_link %}
+  {% set _accessibility_link = pattern('link', {
```
Comment (brummbar) [inline templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig:4]: I am not a fan of using the link pattern, it doesn't support things like url attributes that might be in the url object.
We might leave it for now though.

File: templates/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig [hunk 3]
```diff
@@ -70,6 +70,17 @@
...
   <p> {{ site_owner ? 'This site is managed by the @name'|t({'@name': site_owner}) }} </p>
+
+  {% if accessibility_link %}
+    {% set _accessibility_link = pattern('link', {
+      label: 'Accessibility'|t,
+      path: accessibility_link,
+      standalone: true,
+    }) %}
+    {% block accessibility_link %}
+      <p> {{ _accessibility_link }} </p>
```
Comment (brummbar) [inline templates/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig]: Nit: trim spaces.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig (hunk 2 above)
Comment (drishu) [reply templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig:4]: we resolved that by patching with our bcl_link twig function

File: tests/src/Kernel/FooterBlockTest.php (hunk 1 above)
Comment (drishu) [reply tests/src/Kernel/FooterBlockTest.php:27]: for the usage of the link pattern

**Review by AaronGilMartinez — DISMISSED** [review]

**Review by brummbar — DISMISSED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**